lesson_2/Lesson_2.2.ipynb

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
<ul><li>Sentence missing some words (I don't understand it)</li><li>Image not displaying (may be ReviewNB's fault, in which case ignore my comment)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Fixed the wording. the image doesnt load in reviewnb which is annoying. Its the one on the wikipedia page</p>